frontend/ui/otamanager.lua

```diff
@@ -227,6 +229,10 @@ function OTAManager:fetchAndProcessUpdate()
         UIManager:show(InfoMessage:new{
             text = _("KOReader is up to date."),
         })
+    elseif ota_version == -1 then
+        UIManager:show(InfoMessage:new{
+            text = T(_("Device no longer supported.\n\nPlease check %1"), "https://github.com/koreader/koreader/wiki/deprecated-devices")
```

Comment:
I'm not sure if this change to use template is necessarily a good idea, although of course something like the Chinese documentation on the wiki comes with its own set of issues (mainly that I have no idea how out of date it might be :-) ).

Reply:
Anyway, good to go in my book, please merge today or tomorrow so that there'll be some translations by Saturday.